index.html

```diff
@@ -44,8 +44,138 @@ <h6><a href="">LET'S CONNECT</a></h6>
                 </ul>
             </aside>
         </section>
-       
     </main>
+
+    1
+    <section class = "works">
+        <div class = "cards">
+                <img src = "Images/Tonic-1.png" alt = "own-portfolio-page"/>
```

Comment:
- [ ] I would like to suggest using a background image for this section instead of an `img` tag. This approach would be more suitable as the image is primarily intended for design purposes.
- [ ]  By applying a background image, you can have more flexibility in positioning and styling elements within the section. 🎴